static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -175,40 +176,33 @@ function OnboardingContent({
     };
   });
 
-  const organization = useOrganization();
   const [jsFramework, setJsFramework] = useState<{
     value: PlatformKey;
     label?: ReactNode;
     textValue?: string;
   }>(jsFrameworkSelectOptions[0]);
 
-  const defaultTab =
-    currentProject.platform && backend.includes(currentProject.platform)
-      ? 'jsLoader'
-      : 'npm';
-
-  const {getParamValue: setupMode, setParamValue: setSetupMode} = useUrlParams(
-    'mode',
-    defaultTab
-  );
-
-  const showJsFrameworkInstructions =
-    currentProject.platform &&
-    replayBackendPlatforms.includes(currentProject.platform) &&
-    setupMode() === 'npm';
-
+  const backendPlatform =
+    currentProject.platform && replayBackendPlatforms.includes(currentProject.platform);
+  const mobilePlatform =
+    currentProject.platform && replayMobilePlatforms.includes(currentProject.platform);
   const npmOnlyFramework =
     currentProject.platform &&
     replayFrontendPlatforms
       .filter((p): p is PlatformKey => p !== 'javascript')
       .includes(currentProject.platform);
-
-  const showRadioButtons =
+  const jsLoaderPlatform =
     currentProject.platform &&
     replayJsLoaderInstructionsPlatformList.includes(currentProject.platform);
```

Comment:
Could we simplify that to only check for the presence of the `replayOnboardingJsLoader` key in the doc object?
```
const hasJsLoaderOnboarding = !!docs.replayOnboardingJsLoader
```

Reply:
unfortunately this doesn't work with the radio buttons -- selecting a diff js framework from the dropdown will break the logic  :( just tried it